docs/source/configuration/volto-slate/writing-plugins.md

```diff
@@ -8,18 +8,32 @@ html_meta:
 
 (writing-plugins-label)=
 
-# How to write custom plugins
+# How to write a Slate editor plugin
 
-This section will guide you through registering a custom plugin for `volto-slate`.
+This section will guide you through writing and registering a custom plugin for the Slate editor of Volto.
```

Comment:
In fact Slate is the default editor for text blocks and also for Richtext fields. There are still use cases, even in block based Volto, where you would use schema fields of content types. Richtext fields values are edited per default with the default editor Slate. Do you want to adapt your suggestion?

Reply:
I looked at the Plone 6/Volto demo, and I cannot find a Richtext block in the menu. I only see a Text block, and checked all the folds in the accordion. Am I looking in the wrong place?

![Screen Shot 2022-07-29 at 2 27 18 AM](https://user-images.githubusercontent.com/102112/181729382-51d5273c-2d20-42ab-a9d7-db6c231507da.png)

In the Volto 16 upgrade guide, we stated that migrating from `draftJS` to `slate` was for "text blocks". I don't think we mentioned "Richtext blocks".

This might be a version thing that needs to be clarified for pre-16.

Reply:
Link to Volto Upgrade Guide with v16.0:
https://62e3976a2333690009ffd844--volto.netlify.app/upgrade-guide/index.html#upgrading-to-volto-16-x-x

Reply:
Traditionally Plone has schema based content types. Volto is working with blocks on editing the content of an instance of a content type. Per default (backend integration add-on volto.plone https://github.com/plone/plone.volto installed) the default content types (Page, News Item,…) are 'blocks enabled' (see behaviors in content type control panel). A project can have additional custom content types without blocks enabled but beeing edited field by field the traditional way. Or the project has custom content types with blocks enabled and also fields (shown in the sidebar) which have automagically applied the Slate editor if they are richtext fields (content type schema defines some fields, and some are richtext). The demo does have only default content types and they are 'blocks enabled'.

Reply:
If you want to see this in action on the demo site:
Disable the blocks behavior and enable the richtext behavior for content typ 'News Item'. Then create a News Item. I did it just now on https://6.demo.plone.org/controlpanel/dexterity-types/News%20Item
![image](https://user-images.githubusercontent.com/1144737/181735431-42f27cfd-2143-4c6c-b937-36046812f264.png)

Reply:
Thanks for the explanation. TIL. I updated the suggestion to leave it general, leaving just a change in a preposition.